src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -417,6 +439,30 @@ void ClusterTaskManager::FillResourceUsage(
       by_shape_entry->set_backlog_size(backlog_it->second);
     }
   }
+
+  for (const auto &pair : infeasible_tasks_) {
```

Comment:
Github won't let me leave a comment on line 394, but all the tasks in `tasks_to_schedule_` are now guaranteed to be feasible, so we don't need the if statement there right?